### DISCUSSION TITLE
Different highlight groups overrides for different themes

### DISCUSSION THREAD

**Discussion by gmnz** [NvChad/NvChad#1737] [Q&A] [answered]
Hi,

How can I modify highlight groups like this

```
`M.ui = {
   hl_override = {
      Pmenu = { bg = "white" },

      MyHighlightGroup = {
         fg = "red",
         bg = "darker_black"
      }
   },
}`
```

but do different overrides for different themes, please? I know that I can override specific colors per theme, but I'd rather modify the highlight groups.

**Comment by siduck** [comment, marked as answer]
@gmnz does this work? btw at this moment your feature isnt supported but from v2.0 it will be, i'll work on it, the below example might work tho, but you cant use color names, you have to use raw hex colors. i'll add this feature in v2.0

```lua
M.ui = {

changed_themes = {
      onedark = {
         polish_hl = {
            Normal =  {  bg = "#ffffff"  } 
         }
      }
  }
  
  }
  ```

**Reply by gmnz** [reply]
This works perfectly, thanks a lot! :-)

**Reply by siduck** [reply]
@gmnz docs for it https://nvchad.com/config/theming#override-specific-colors-in-themes 

Also the changed_theme table just merges the onedark table to default onedark table in base46 repo, polish_hl table is present in many base46 themes, example https://github.com/NvChad/base46/blob/master/lua/base46/themes/monekai.lua#L55 

its used to override hlgroup colors specially for that theme so the solution above worked!
